errors/build.gradle

```diff
@@ -33,13 +33,32 @@ android {
     kotlinOptions {
         jvmTarget = '1.8'
     }
+
+    testOptions {
+        unitTests {
+            includeAndroidResources = true
+        }
+    }
+
+    buildFeatures {
+        viewBinding true
+    }
 }
 
 dependencies {
+    implementation project(path: ':strings')
+    implementation project(path: ':androidshared')
+
     implementation Dependencies.androidx_core_ktx
     implementation Dependencies.androidx_appcompat
     implementation Dependencies.androidx_recyclerview
     implementation Dependencies.android_material
-    implementation project(path: ':strings')
-    implementation project(path: ':androidshared')
+
+    testImplementation project(path: ':androidtest')
+    testImplementation project(path: ':testshared')
+
+    testImplementation Dependencies.androidx_test_ext_junit
+    testImplementation Dependencies.androidx_test_espresso_core
+    testImplementation Dependencies.androidx_test_espresso_contrib
+    testImplementation Dependencies.robolectric
```

Comment:
I don't think you need the `contrib` or the `robolectric` dependency here.

Reply:
Fixed.